index.html

```diff
@@ -182,13 +288,41 @@ <h3>Multi-Post Stories</h3>
     <li class="skill">ruby</li>
  </ul>
  
-    <button class="btn" type="button" >  See Project</button>
+    <button class="btn" type="button"> See Project </button>
+             </div>
+ </div> 
+ 
+ <div class="work2-D">
+     
+     <div class="image2"> 
+         <img src="Screenshot(349).png" alt="">
+      </div>
   
-</div>
+         <div class="story-D-2">
+  
+     <h3>Multi-Post Stories</h3>
+     <p class="text">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text. </p>
+  <ul class="skil">
+     <li class="skill">css</li>
+     <li class="vector">.</li>
+     <li class="skill">html</li>
+     <li class="vector">.</li>
+     <li class="skill">bootstrap</li>
+     <li class="vector">.</li>
+     <li class="skill">ruby</li>
+  </ul>
+  
+     <button class="btn" type="button"> See Project </button>
```

Comment:
- [ ] This is awesome.. Just a little change 😃  - The color for the `See project text` should also match the one specified in your design. The goal is for us to come up with apps that look exactly like the designed [mockups](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A976).